src/util/records.py

```diff
@@ -782,8 +782,8 @@ def rename_xmlid(cr, old, new, noupdate=None, on_collision="fail"):
         if model == "ir.ui.view" and column_exists(cr, "ir_ui_view", "key"):
             cr.execute("UPDATE ir_ui_view SET key=%s WHERE id=%s AND key=%s", [new, new_id, old])
             if cr.rowcount:
-                # iif the key has been updated for this view, also update it for all other cowed views.
-                # Don't change the view keys inconditionally to avoid changing unrelated views.
+                # if the key has been updated for this view, also update it for all other cowed views.
```

Comment:
This is not a typo, it's a [common](https://en.wikipedia.org/wiki/If_and_only_if) shorthand of `if and only if` in logic predicates.

Reply:
It should then written `iff`

Reply:
Yes, it should.